docs/api/ReactWrapper/state.md

```diff
@@ -25,3 +25,5 @@ expect(wrapper.state('foo')).to.equal(10);
 
 - [`.props() => Object`](props.md)
 - [`.prop(key) => Any`](prop.md)
+- [`.context() => Object`](context.md)
+- [`.context(key) => Any`](context.md)
```

Comment:
you might want to follow what `.state()` does and instead just have `.context([key]) => Any`